CHANGELOG-fix-changelog-entry.md

```diff
@@ -0,0 +1 @@
+- Fix changelog formatting for v0.101.1 entries.
```

Comment:
This changelog has already been compiled and needs to be removed - sorry this is a pain.